connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -570,3 +571,21 @@ message Hint {
   // (Optional) Hint parameters.
   repeated Expression.Literal parameters = 3;
 }
+
+// Unpivot a DataFrame from wide format to long format, optionally leaving identifier columns set.
+message Melt {
```

Comment:
since `melt` == `unpivot`, and `Unpivot` is just a logical plan, so I suggest to rename it `Unpivot`